src/main/java/edu/wpi/grip/ui/PaletteView.java

```diff
@@ -49,6 +105,12 @@ public void clearOperations() {
 
     @Subscribe
     public void onOperationAdded(OperationAddedEvent event) {
-        this.operations.getChildren().add(new OperationView(this.eventBus, event.getOperation()));
+        OperationView view = new OperationView(this.eventBus, event.getOperation());
+        this.operations.getChildren().add(view);
+        FXCollections.sort(this.operations.getChildren(),
```

Comment:
If we're going to sort all operations by name, we have to add categories so the groups of related operations don't get scrambled up